### DISCUSSION TITLE
Welcome to SiEPIC_EBeam_PDK Discussions!

### DISCUSSION THREAD

**Discussion by lukasc-ubc** [SiEPIC/SiEPIC_EBeam_PDK#230] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Likynti25** [comment]
Hello Prof. Lukas, my question is, "how can we add PIN to the design circuit in Klayout"? 
Thank you

**Comment by mustafacc** [comment]
Hi @Likynti25,

The SiEPIC EBeam PDK fabrication process does not currently support P and N implantations and therefore this device is not part of the library.

Other process you can try that have this capability is the AMF process for which there's a confidential SiEPIC_AMF_PDK for that you can access through:
https://www.cmc.ca/amf-silicon-photonics-general-purpose/

**Comment by Likynti25** [comment]
Thank you very much for your response.

**Comment by frankiejxc** [comment]
Dear  Prof. Lukas, do you know which website in github could provide the free silicon photonic PDK?
